src/main/java/org/opentripplanner/updater/vehicle_positions/PollingVehiclePositionUpdater.java

```diff
@@ -0,0 +1,142 @@
+package org.opentripplanner.updater.vehicle_positions;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.transit.realtime.GtfsRealtime.VehiclePosition;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.updater.GraphUpdaterManager;
+import org.opentripplanner.updater.GtfsRealtimeFuzzyTripMatcher;
+import org.opentripplanner.updater.JsonConfigurable;
+import org.opentripplanner.updater.PollingGraphUpdater;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * Add vehicle positions to OTP patterns via a realtime source
+ **
+ * <pre>
+ * rt.type = vehicle-position-updater
+ * rt.frequencySec = 60
+ * rt.sourceType = gtfs-http
+ * rt.url = http://host.tld/path
+ * rt.feedId = TA
+ * </pre>
+ *
+ */
+public class PollingVehiclePositionUpdater extends PollingGraphUpdater {
+    private static final Logger LOG = LoggerFactory.getLogger(org.opentripplanner.updater.vehicle_positions.PollingVehiclePositionUpdater.class);
```

Comment:
Could this just be:

```suggestion
    private static final Logger LOG = LoggerFactory.getLogger(PollingVehiclePositionUpdater.class);
```

Reply:
Don't know how hat slipped in there. Thanks for noticing! Fixed in 1360e72e44678135324c29f7ac4942514fa05270